azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerPropertyResolutionTest.java

```diff
@@ -67,6 +68,78 @@ public void testPropertyResolution() throws Exception {
     assertProperties(false);
   }
 
+  @Test
+  public void testNodeOverrides() throws Exception {
+    this.testUtil = new FlowRunnerTestUtil(EXEC_FLOW_DIR, this.temporaryFolder);
+
+    final HashMap<String, String> flowProps = new HashMap<>();
+    flowProps.put("props7", "execflow7");
+    flowProps.put("props6", "execflow6");
+    flowProps.put("props5", "execflow5");
+
+    // enable overriding also for existing job props
+    final FlowRunner runner = this.testUtil.createFromFlowMap(FLOW_NAME, null, flowProps,
+        Props.of(ConfigurationKeys.EXECUTOR_PROPS_RESOLVE_OVERRIDE_EXISTING_ENABLED, "true"));
```

Comment:
Can we please set `EXECUTOR_PROPS_RESOLVE_OVERRIDE_EXISTING_ENABLED` to false (the default) for all tests? In my opinion that's the best use for the tests since with the new _flowOverride_ capabilities every property at any level will be overwritten. We can create separate tests for when the flag is set to true.

Reply:
With https://github.com/azkaban/azkaban/pull/2597/commits/6b5b07687a0119aa168492358e1ec06dccd72a12 both modes are now tested equally.

Reply:
Thanks!